app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2538,8 +2538,8 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
         disable_navigator <- any_(&set_navigator_false,&some_vis_selected);
         enable_navigator  <- any_(&set_navigator_true,&no_vis_selected);
 
-        eval_ disable_navigator ( model.navigator.disable() );
-        eval_ enable_navigator  ( model.navigator.enable()  );
+        eval_ disable_navigator ( model.navigator.settings().disable() );
+        eval_ enable_navigator  ( model.navigator.settings().enable()  );
```

Comment:
This makes the API uglier. I want to disable the navigator, not the navigator settings! YOu can refactor the internal structure, but keep the API nice.

Reply:
Fixed. I didn't change `navigator.settings().set_max_zoom(...)` API though, because I feel it is sensible.